src/plugins/controls/timer/component/scss/timerbox.scss

```diff
@@ -8,6 +8,10 @@
         @include transition(opacity, 300ms, linear);
     }
 
+    & #timerbox-aria-label {
+        display:none;
```

Comment:
Space after colon. Also display none make element invisible for the screen readers, does it make a sense?

Reply:
used to provide data in another plugin via `aria-labelledby`